wordpress-mobile/WordPress-iOS
# Clear warnings on XCode 9.3

**Fixes** #9027

**To test:**

1. Clean the project.
2. Build the project and check that there's only 2 warnings: one in HelpshiftUtils.m and the "Some object fields have incompatible Objective-C category definitions".
3. Test the app does not explode.

cc: @diegoreymendez
